forms-flow-web/src/components/Form/List.js

```diff
@@ -166,11 +166,18 @@ const List = React.memo((props) => {
     formType
   ]);
 
-
+   
+  const checkForm = ()=>{
+     return formCheckList.length === 1 ? t("Form") : t("Forms");
+    };
+  const checkResource = ()=>{
+     return  formCheckList.length === 1 ? t("Resource") : t("Resources");
+    };
+    
   const downloadForms = () => {
-    FileService.downloadFile({ forms: formCheckList }, () => {
+    FileService.downloadFile({ forms: formCheckList}, formType , () => {
       toast.success(
-        `${formCheckList.length} ${formCheckList.length === 1 ? t("Form") : t("Forms")
+        `${formCheckList.length} ${(formType === "form") ? checkForm() : checkResource()
```

Comment:
` ${ formCheckList.length ==1? t(formType) : t(`${ formType}s`)}`

Reply:
no need to write more functions here it will cause to call stack limit to exceed like RangeError

Reply:
Do we really need this? There can be situation of selecting resources and forms together to migrate?

Reply:
we are selecting the forms and resources seperatly .Thinks needed

Reply:
Is the code adds it seperately or reset the list of selected forms on switching?

Reply:
Here the form download data only deferens at type